test-e2e/manager-basic.js

```diff
@@ -56,20 +59,90 @@ test('Managing created projects', async (t) => {
   t.ok(project1)
   t.ok(project2)
 
-  t.test('initial settings from project instances', async (st) => {
-    const settings1 = await project1.$getProjectSettings()
-    const settings2 = await project2.$getProjectSettings()
-
-    st.alike(settings1, {
-      name: undefined,
-      defaultPresets: undefined,
-    })
+  await t.test(
+    'initial settings and default config from project instances',
+    async (st) => {
+      const settings1 = await project1.$getProjectSettings()
+      const settings2 = await project2.$getProjectSettings()
+
+      const {
+        presets: defaultPresets,
+        fields: defaultFields,
+        icons: defaultIcons,
+      } = await getExpectedConfig(
+        new URL('../config/defaultConfig.mapeoconfig', import.meta.url).pathname
+      )
+
+      st.is(settings1.name, undefined, 'undefined name for project1')
+      st.is(settings2.name, 'project 2', 'matched name for project2')
+
+      const projectPresets = await project1.preset.getMany()
+
+      st.is(
+        settings1.defaultPresets?.point.length,
+        defaultPresets.length,
+        'the default presets loaded is equal to the number of presets in the default config'
+      )
+
+      st.alike(
+        projectPresets.map((preset) => preset.name),
+        defaultPresets.map((preset) => preset.value.name),
+        'project is loading the default presets correctly'
+      )
+
+      const projectFields = await project1.field.getMany()
+
+      st.alike(
+        projectFields.map((field) => field.tagKey),
+        defaultFields.map((field) => field.value.tagKey),
+        'project is loading the default fields correctly'
+      )
+
+      const projectIcons = await project1[kDataTypes].icon.getMany()
```

Comment:
maybe read icons from presets?

Reply:
How would I do that?
I mean, I can do `project1.presets.getMany()` and from that get all the `iconId`s, but that wouldn't be of much help since the `expected.icons` are not records on the db, with a `docId` that I can match